webui/src/docs/rest-api.jsx

```diff
@@ -1,244 +1,540 @@
+            consists of a set of common fixed metadata fields and a set of custom
+            metadata blocks, each block containing related metadata fields. A
+            record is always connected to one scientific community which has the
+            role of curating and maintaining it.</p>
+        <p> A data record can exist in several states. Immediately after creation a record
+            enters the 'draft' state. In this state the record is only accessible
+            by its owner and can be freely modified: its metadata can be changed
+            and files can be uploaded into or removed from it. A draft can be
+            published at any time, and through this action it changes its state
+            from 'draft' to 'published', is assigned Persistent Identifiers,
+            and becomes publicly accessible. <strong>Please note that the list
+            of files in a <em>published record</em> cannot be changed</strong>.
+            </p>
+        <p> A record contains a set of common metadata fields and a set of
+            custom metadata blocks. This metadata is not free form, however,
+            but is governed by static schemas; the common metadata schema is
+            set by B2SHARE and defines a superset of Dublin Core elements,
+            while the schema for the custom metadata block is specific to each
+            community and can be customized by the community administrators.
+            The schemas are formally defined in the JSON Schema format. A
+            special HTTP API call is available for retrieving the JSON Schema
+            of a record in a specific community. In order to be accepted, the
+            records submitted to a community must conform to the schema
+            required by the community.
+        </p>
+
+        <h3>Authentication</h3>
+        <p>Only authenticated users can use the API. Each HTTP request to the
+            server must pass an <code>access_token</code> parameter that
+            identifies the user. The <code>access_token</code> is an
+            opaque string which can be created in the user profile when
+            logged in to the B2SHARE web user interface. B2SHARE’s access
+            tokens follow the OAuth 2.0 standard. </p>
+        <p>To get an access token, login to the B2SHARE web interface and click
+            on your username. On the User Profile page, go to the “API Tokens”
+            section, enter a token identification name (e.g. 'api') and click
+            “New Token”. This will create an access token, visible on the
+            screen. Please note that this is the only time the access token is
+            visible, so copy it to a safe place. </p>
+        <p>The following shell commands will expect that the ACCESS_TOKEN
+            environment variable is defined and contains the actual
+            access_token. The command to define this variable is: </p>
+        <Example>export ACCESS_TOKEN='...'</Example>
```

Comment:
I would suggest to put a real access-token (not valid of course). The users will most probably know how to export env variables as they want to use the REST API. Thus this example would be more useful if it also showed what an access token looks like.